content/metadata/2022/menu.yml

```diff
@@ -38,6 +38,8 @@
 - name: Info & Support
   url: info.html
   submenu:
+  - name: Venue
```

Comment:
Could you please also add this to the info.md file?

Reply:
done